src/heading.js

```diff
@@ -27,46 +25,120 @@ export default class Heading extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ HeadingEngine ];
+		return [ Paragraph, HeadingEngine ];
 	}
 
 	/**
 	 * @inheritDoc
 	 */
 	init() {
 		const editor = this.editor;
-		const command = editor.commands.get( 'heading' );
-		const options = command.options;
-		const collection = new Collection();
-
-		// Add options to collection.
-		for ( const { id, label } of options ) {
-			collection.add( new Model( {
-				id, label
+		const dropdownItems = new Collection();
+		const options = this._getLocalizedOptions();
+		const commands = [];
+		let defaultOption;
+
+		for ( let option of options ) {
+			// Add the option to the collection.
+			dropdownItems.add( new Model( {
```

Comment:
OTOH I'm not sure whether would make more sense. There could be cases when a feature using the dropdown might want to play with models in `#items` immediately, i.e. bind its attributes to something else.

If we enforce that `#items` is `Collection.<Object>`, developers would need to pick those models outside of the dropdown after `createListDropdown` which is not very convenient. 

Besides, what would `ListView` look like then if `#items` would be `Collection.<Object>`?. Where to store the "intermediate models" created in `createListDropdown()`? Because there **must** be models for `Model->ListItemView` bindings.